src/components/IssueSearchAdvanced.vue

```diff
@@ -23,6 +25,8 @@ watch(isNonEmptyDescriptionSelected, () => {
   const facet = facets.value.find(facet => facet.field === 'cve_description');
   if (isNonEmptyDescriptionSelected.value) {
     if (!facet) {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      //@ts-ignore
       facets.value.push({ field: 'cve_description', value: 'nonempty' });
```

Comment:
I think we can rewrite the `facet`  bit so we don't need this.
Maybe using `range?` on the facet type and then asserting that `range` exists on `parseValueForURL`